server/server.ts

```diff
@@ -3,6 +3,11 @@ import cors from 'cors';
 import cookieParser from 'cookie-parser';
 import path from 'path';
 
+// import { userQueryController } from './controllers/userQueryController'
+import { openaiNLPQuery } from './controllers/openaiNLPQuery.ts';
+import { googlePlacesApi } from './controllers/googlePlacesApi.ts';
```

Comment:
I have a couple of middleware functions exported from this controller, so we will need to import them specifically here. But this is a good skeleton